PhysicsTools/PatAlgos/plugins/PATPackedCandidateProducer.cc

```diff
@@ -266,6 +266,8 @@ void pat::PATPackedCandidateProducer::produce(edm::StreamID, edm::Event& iEvent,
 	     whiteList.find(ic)!=whiteList.end() || 
              (cand.trackRef().isNonnull() &&  whiteListTk.find(cand.trackRef())!=whiteListTk.end())
 	    ) {
+	   outPtrP->back().setFirstHit(ctrack->hitPattern().getHitPattern(reco::HitPattern::TRACK_HITS, 0));
+
```

Comment:
github apparently doesn't parse tabs vs spaces well and this shows up with incorrect indentation on the web.
It would be nice to replace tabs with spaces here